pkg/config/config.go

```diff
@@ -729,7 +732,7 @@ func (c *config) Validate(fsys fs.FS) error {
 		c.Db.Image = pg13
 	case 14:
 		c.Db.Image = pg14
-	case 15:
+	case 15, 17:
```

Comment:
Let's add a new case for pg17 instead. No need to touch orioledb for the time being.

```
	case 17:
		c.Db.Image = pg17
```